test/windows/acceptance/logstash_release_default_plugins.ps1

```diff
@@ -0,0 +1,30 @@
+# Created By: Gabriel Moskovicz
+#
+# To be run on Jenkins
+#
+# Requirements to run the test:
+# 
+# - Powershell 4
+# - Windows 7 or newer
+# - Java 7 or newer
+# - Ruby 7 or newer
+
+$ruby = $env:RUBY_HOME  + "\jruby.exe"
+
+$env:SPEC_OPTS="--format documentation"
```

Comment:
Lets use the default format of rspec that output "dots", people can always override the `SPEC_OPTS` variable if they need more verbosity.